gui_agents/agents/Backend/LybicSDKBackend.py

```
@@ -0,0 +1,355 @@
+        
+        # 初始化SDK客户端（仅在有必要参数时）
+        if self.api_key and self.org_id:
+            self.client = LybicClient(
+                org_id=self.org_id,
+                api_key=self.api_key,
+                endpoint=self.endpoint,
+                timeout=self.timeout,
+                extra_headers=self.extra_headers or {}
+            )
+        else:
+            raise ValueError("LYBIC_API_KEY and LYBIC_ORG_ID are required. Please set them as environment variables or pass them as arguments.")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)